src/app/pages/Languages.svelte

```diff
@@ -167,7 +167,7 @@
       </a>
       <header class="languages__container--header">
         <div>
-          <h1>{$_('common.name')}</h1>
+          <h1>{$_('common.app_name')}</h1>
           <p>designed for
```

Comment:
Do we want to replace "designed for" with the following?
```suggestion
          <p>{$_('page.main.designed_for')}
```

Reply:
Good catch!